scripts/lint.py

```diff
@@ -148,12 +154,12 @@ def lint_file(filepath: str) -> int:
         root_dirpath = os.path.abspath(f"{script_dirpath}/..")
         os.chdir(root_dirpath)
 
-        extensions = ["html", "js", "py", "rs", "sh", "toml", "wgsl", "yml"]
+        extensions = ["html", "js", "md", "py", "rs", "sh", "toml", "wgsl", "yml"]
 
         exclude_dirs = {"env", "venv", "target", "target_ra", "target_wasm"}
 
         exclude_paths = {
-            "./CONTRIBUTING.md",
+            "./CODE_STYLE.md",
```

Comment:
why would CONTRIBUTING.md be spared tho?

Reply:
CODE_STYLE mentions how NOT to write `TODO`, and so it triggers those lints

Reply:
oooo it's an _exclude_, I see